network-api/networkapi/buyersguide/templates/catalog.html

```diff
@@ -77,10 +91,19 @@ <h1 class="h1-heading text-center mt-0">{% trans "Be Smart. Shop Safe." %}</h1>
               <div class="product-name body">{{product.name}}</div>
             </a>
           </figcaption>
+
+          <input type="hidden" class="product-blurb" value="{{ product.blurb }}">
```

Comment:
HTML validation errors:

<img width="992" alt="image" src="https://user-images.githubusercontent.com/2896608/97638620-2c426800-19fa-11eb-95b1-665ec9660949.png">

Maybe we can have the info as `data-` attributes instead of creating DOM nodes to hold it? Or move the two `<input>` to within `<figcapton>.

Reply:
That seems... not true? `<input type="hidden">` isn't actually an input element.

Reply:
hm, actually, let me try putting them inside the figcaption...